### ISSUE TITLE
Balancing Update: Horse Archers

### ISSUE THREAD

**Issue by GRhin** [Sh0wdown/UnofficialCrusaderPatch#368]
This is mostly a response to finally playing online multiplayer, the games end up with everyone using the same strategy in all circumstances, I see this as unbalanced. There are two units I would like to see changed slightly to fix this.
**This thread is about Horse Archers.** From my research in online forums about this, playersdont like shields OP (another thread created for that), however when they say "No Shields" Horse Archers become more of a problem as they just get spammed so you have 2-300 horse archers coming at you.
Now, I think HA are OK balanced, the problem here is the numbers. spending 10k gold on HA will clear any defense, even with 10-15k gold spent on defenders. This isnt right. Defense should be more powerful from a logical standpoint (as you have the high ground) and from a balancing standpoint (attackers have seige to help as well).
**Therefore I suggest increasing the cost of HA to something like 150 gold** or more (maybe some testing could fine-tune this number somewhat)

**Comment by ilosevsrat** [comment]
Totally agreed, ha are op their cost must be increased 115 gold minimum, you can just spam them and clear any defence, specially verus ai,

**Comment by Halbarad1** [comment]
Agree! Horse archers are extremely powerful. So we should raise cost and probably also put their defense down.

**Comment by Mizo5119** [comment]
You can stop playing death match online and just go for normal mode. That also helps in balancing merc units.

**Comment by Halbarad1** [comment]
@Mizo5119 No that doesn't solve it. In Stronghold Crusader, a player can build up his economy really fast. So in that case, the problem maybe comes up a bit later, but it definitely comes up (except players get killed before they finished their castles). 
Or do you really think that players would stop building them if they would cost even 150 gold?
Also it would be logical that horsemen would cost much more than a normal soldier.

**Comment by Mizo5119** [comment]
I dont disagree with the nerf, but online games tend to always play with 10k gold startup and some nice 10-20 mins of peace. No wonder how players develop their economies very quickly and start cheese snowballing with their "casltles" ( more like factory stimulators).

Horse Archers could use a price up though.

**Comment by GRhin** [comment]
Thanks for the support guys. 
Mizo, as you are the only one to disagree (or at least hesitant about this change) I shal respond to your messages in particular.
Game mode is irrelevant, as you build up massive economies that pay for the HA, I have not seen a single online match where the players who did well used any other unit to attack with. And thats the problem I want to get fixed.
Every match follows the same pattern:
1. economy boom.
2. spam HA, shields, assassins, and maybe knights
3. take HA and shields out to the enemy army.
4. use assassins or knights to try kill the sheilds. 
5. winner carries on to enemy base
6. build cats or use your assasins/knights to destroy wall.
7. bring in a cheap unit to dig up moat
8. kill enemy lord.

of course steps 2-4 often get repeated several times untill someone goes bankrupt and is unable to replace their HA.
If you remove shields from the mix cos they are OP (other thread) you just removed the nessesity of assasins and knights. 
So there are in most games maybe 6 unit types used by all players.  the other 20-odd unit types dont get used, and are never a good idea, that imho is the definition of an unbalanced unit setup.
However if ha were alot more expensive, then sometimes it might be a better idea to recruit archers or crossbows in bulk instead of ha, and if that were the case, macemen and spears become more practical to hit them. If sheilds were also weaker youd want to include macemen and spears in your assault to deal with enemy archers, then you might want to include some pikemen or swordsmen to better defend your archers from the enemy maces and spears. 
also, as it becomes harder to remove enemy from their walls, defensive strategys are more practical, and castle design is more important, and therefore other seige units also become more important. Use seige towers to get your swords up the walls, or ladders to get your archers onto an isolated tower for the height perks (though for some reason crenelations prevent these two from working for me, maybe i should create another ticket for that? or is it just me?)

**Comment by Starshine575** [comment]
@GRhin I really like your idea of making HA more expansive, because, as already mentioned, they are ATM OP and also very Overused. ✔😮

However, I don't agree with the "stable building for HA". Arabian troops only cost gold and that is a absolutly good thing in my opinion. ❌

**Comment by GRhin** [comment]
Agree, was just trying to float alternative ideas.

**Comment by Ariel644** [comment]
Agree, & maybe All of the mercenaries unit cost is increased

**Comment by GRhin** [comment]
there is an arguement for that, i dont disagree, but i think i would have to test the balance with just the HA increased to be able to form an opinion on the cost of the rest of the units. slaves for one i think are priced right (they die so quick, increasing that cost makes them unviable), Arabian archers I personally think are priced high enough to make them almost overpriced compared to European archers (though I believe some other players disagree), slingers are pretty pointless already, and i dont think their cost has anything to do with that.
